### PR TITLE
responsive searchbar

### PR DESCRIPTION
**Pull request by shubhampr07**
Now, the Search Bar is fully responsive in every screen size.

### CODE REVIEW

**Comment by Yana-Gupta** [comment]
@shubhampr07 deployment is failing, make sure there is no unused variable because in deployment we have to remove unused variables, check [here](https://app.netlify.com/sites/symphonious-khapse-a9fcba/deploys/651c3b03b5e662000817c1a8)

**Comment by shubhampr07** [comment]
made all the changes, check once

File: src/components/Home/Index.jsx
Comment (Yana-Gupta) [inline src/components/Home/Index.jsx:1]: Well may I ask the reason to replace `doubleQuotes` to `singleQuotes`. Any specific reason? If not please change it to `doubleQuotes`

Comment (shubhampr07) [reply src/components/Home/Index.jsx:1]: sorry i haven't noticed that . Now, Done

**Comment by Yana-Gupta** [comment]
@shubhampr07 there are merge conflicts solve them.

**Comment by Yana-Gupta** [comment]
Thanks @shubhampr07 🥇

**Pull request merged**